charts/portworx/templates/portworx-k8s-secrets.yaml

```diff
@@ -0,0 +1,32 @@
+{{- $secretType := .Values.secretType | default "k8s" }}
+
+{{- if eq $secretType "k8s" }}
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: portworx
+---
+kind: Role
+apiVersion: rbac.authorization.k8s.io/v1
+metadata:
+  name: px-role
+  namespace: portworx
+rules:
+- apiGroups: [""]
+  resources: ["secrets"]
+  verbs: ["get", "list", "create", "update", "patch"]
+---
+kind: RoleBinding
+apiVersion: rbac.authorization.k8s.io/v1
```

Comment:
Don't we have a global flag for the RBAC version?
 (afaik RBAC version changes depending on Kubernetes version)

Reply:
Good catch @zoxpx . I have used the global template for it now.